### PR TITLE
feat(i18n): add Vietnamese with 100% coverage

### PR DESCRIPTION
**Pull request by hnxb**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [X] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [X] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [X] Feature
-   [ ] Refactoring (no functional changes, no api changes)
-   [ ] Other... Please describe:

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [X] No

### CODE REVIEW

File: projects/i18n/languages/vietnamese/core.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import {LanguageCore} from '@taiga-ui/i18n/interfaces';
+import {TUI_VIETNAMESE_LANGUAGE_COUNTRIES} from './countries';
+
+export const TUI_VIETNAMESE_LANGUAGE_CORE: LanguageCore = {
+    months: [
+        'Tháng 1',
+        'Tháng 2',
```
Comment (nsbarsukov) [inline projects/i18n/languages/vietnamese/core.ts:7]: Sorry if this is a stupid question.
I know nothing in vietnamese language. That is why i am surprised by this namings of months.

The names of months are just "month" in vietnamese + index of the month, are not they ? Does months have more contrast names like "January", "February", and etc ?

Comment (hnxb) [reply projects/i18n/languages/vietnamese/core.ts:7]: Hi,

Yes, we use 'month + index' for every month in a year, but there's another name for January and December that are "tháng giêng" and "tháng chạp".

Comment (nsbarsukov) [reply projects/i18n/languages/vietnamese/core.ts:7]: got it, thanks 👌

**Review by nsbarsukov — APPROVED** [review]

**Review by vladimirpotekhin — APPROVED** [review]

**Review by waterplea — APPROVED** [review]

**Comment by waterplea** [comment]
Thank you, @hnxb !

**Pull request merged**